commands/Site_Remove_Atlantis_Legacy_Modules.php

```
@@ -0,0 +1,1566 @@
+
+				// Initialize repository (handle deployment configuration errors gracefully).
+				try {
+					$this->initialize_repository( $input, $output );
+				} catch ( \Exception $e ) {
+					// Check if it's a deployment configuration error (DeployHQ for Pressable, WPCOM GitHub Deployments for Atomic).
+					$is_deployment_error = str_contains( $e->getMessage(), 'WPCOM GitHub Deployments' )
+						|| str_contains( $e->getMessage(), 'GitHub repository' )
+						|| str_contains( $e->getMessage(), 'DeployHQ' );
+
+					if ( $is_deployment_error ) {
+						$deployment_type = ( 'pressable' === $site_host ) ? 'DeployHQ' : 'WPCOM GitHub';
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$site_host'. (undefined)

(UnusedLocalVariable)